examples/placeholder/linux/apps/app1/config.matter

```diff
@@ -2999,6 +3062,21 @@ endpoint 0 {
     callback attribute clusterRevision default = 1;
   }
 
+  server cluster AccessControl {
+    emits event AccessControlEntryChanged;
+    emits event AccessControlExtensionChanged;
```

Comment:
This is wrong.  This does not support the Extension attribute, so does not in fact emit this event.  @krypton36